app/Http/Controllers/Admin/OrderListController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Services\Customer\OrderService;
+use Illuminate\Http\Request;
+
+class OrderListController extends Controller
+{
+    private $orderService;
+
+    /**
+     * @param $customerService
+     */
+    public function __construct(OrderService $orderService)
+    {
+        $this->orderService = $orderService;
+    }
+
+    public function index()
+    {
+        $orders = $this->orderService->getListOrder();
+        //If get date ... to date ... then filter products by date
+        if (request()->date_from && request()->date_to) {
+            $orders = $this->orderService->getListOrderByDate(request()->date_from, request()->date_to);
+        }
+        return view('admin.order.listorder', ['orders' => $orders]);
+    }
+
+    public function store(Request $request)
+    {
+        $key = $request->search;
+        $search = '%' . $key . '%';
+        // if you receive a request of search type, then conduct a search
+        switch ($request->search_type) {
+            case 1:
+                $orders = $this->orderService->searchOrderByCustomerName($search);
+                break;
+            case 2:
+                $orders = $this->orderService->searchOrderHaveProductName($key);
+                break;
+            default:
+                $orders = $this->orderService->getListOrder();
+                break;
+        }
+        return view('admin.order.listorder', ['orders' => $orders]);
+    }
+
+    public function edit($id)
+    {
+        $order = $this->orderService->findById($id);
+        return view('admin.order.edit', ['order' => $order]);
+    }
+
+    public function update($id,Request $request)
+    {
+        $this->orderService->updateStatus($id,$request);
+        return redirect()->route('admin.order.index');
```

Comment:
you need the result of update action to response

Reply:
I added a return message